zha/application/platforms/cover/__init__.py

```diff
@@ -123,6 +116,10 @@ async def async_stop_cover(self, **kwargs: Any) -> None:
 class Cover(BaseCover):
     """Representation of a ZHA cover."""
 
+    DEFAULT_MOVEMENT_TIMEOUT: float = 5
+    LIFT_MOVEMENT_TIMEOUT_RANGE: float = 300
+    TILT_MOVEMENT_TIMEOUT_RANGE: float = 30
```

Comment:
This doesn't really need to be in the class, right?
Seems a bit weird to have constants (in caps) as class attributes.

Reply:
Fair enough, thanks for the amendments @puddly